src/strategies/mcn-farm/index.ts

```diff
@@ -0,0 +1,117 @@
+import { formatUnits } from '@ethersproject/units';
+import { call, multicall } from '../../utils';
+import { BigNumber } from '@ethersproject/bignumber';
+
+export const author = 'drop-out-dev';
+export const version = '0.1.0';
+
+const FARM_ADDRESS = '0x15dEd15fE32EBac0b6cFb08cdAB112cca8380423';
+const MCN_ADDRESS = '0xD91E9a0fEf7C0fa4EBdAF4d0aCF55888949A2a9b';
+const MCN_LP_ADDRESS = '0x2Ef2cb6af83de4171A69EE2f7C677079fFD9BcD0';
```

Comment:
would it be good to have these addresses passed from params? (but also still have these as defaults) so others can pass their own addresses if they want to, or it doesn't make sense in this strategy?

Reply:
Yes, good idea. I've updated accordingly.